manifest.php

```diff
@@ -24,7 +24,7 @@
     'label'       => 'Test Booklets',
     'description' => 'An extension for TAO to create test booklets (publishable in MS-Word and PDF along with Answer Sheets)',
     'license'     => 'GPL-2.0',
-    'version'     => '1.10.1',
+    'version'     => '1.10.2',
```

Comment:
Since it is an additional feature, it would be better to increase the minor version instead.